src/main/python/ayab/scene.py

```diff
@@ -90,12 +89,16 @@ def refresh(self):
 
         # draw "machine"
         rect_orange = QGraphicsRectItem(
-            -machine_width / 2 - self.LIMIT_BAR_WIDTH, -self.BAR_HEIGHT,
-            machine_width / 2 + self.LIMIT_BAR_WIDTH, self.BAR_HEIGHT)
+            -machine_width / 2 - 0.5,
```

Comment:
Any chance to get rid of these magic numbers?

Reply:
sure